src/main/java/microsoft/exchange/webservices/data/EwsXmlReader.java

```diff
@@ -422,7 +422,7 @@ public String readValue() throws XMLStreamException,
     } else if (this.presentEvent.getEventType() == XmlNodeType.CHARACTERS
         && this.presentEvent.isCharacters()) {
                         /*
-			 * if(this.presentEvent.asCharacters().getData().equals("<")) {
```

Comment:
There are some errors with project formatter.
- Comment line formatting:
 - IntelliJ: does not format comment line indentation. So, every indentation of the code above is right based on your formatter.
 - Eclipse: formats (or changes) comment line indentation.
- google-styleguide for Java works different for Eclipse and Intellij.
- Mix of space and tab are being used as comment indentation character.
 - A tab could be a different number of columns depending on your environment, but a space is always one column.
 - In terms of how many spaces (or tabs) constitutes indentation, it's more important to be consistent throughout your code than to use any specific tab stop value.